pom.xml

```diff
@@ -2,14 +2,14 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.thoughtworks.gauge.maven</groupId>
     <artifactId>gauge-maven-plugin</artifactId>
-    <version>1.6.1</version>
+    <version>1.6.2</version>
     <packaging>maven-plugin</packaging>
     <name>Gauge Maven Plugin</name>
     <url>http://github.com/getgauge/gauge-maven-plugin</url>
     <description>A maven plugin to execute gauge specs in the project</description>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.version>3.8.1</maven.version>
+        <maven.version>4.0.0-alpha-12</maven.version>
```

Comment:
I don't really understand how this solves the problem but we shouldnt depend on an unreleased alpha version of Maven.

`3.9.6` is the latest Maven release.